README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
```

Comment:
Added backticks to `uswds` in 6436fe4 rather than capitalizing to match how we talk about it in the code.